### PR TITLE
update s2i docs

### PR DESCRIPTION
**Pull request by soulseen**
Signed-off-by: zhuxiaoyang <sunzhu@yunify.com>

**What type of PR is this?**

/kind documentation

/cc @runzexia @FeynmanZhou

### CODE REVIEW

File: docs/en/concepts-and-designs/s2i_b2i_overview.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## What is S2I/B2I
+
+Kubesphere S2I(Source-to-Image) / B2I(Binary-to-Image) provides easy CI/CD functionalities, help developers to publish their service to Kubernetes without writing Dockefile.
+
+## Workflow Example
+
+![](https://pek3b.qingstor.com/kubesphere-docs/png/20191223103533.png)
+
+## Principle
```
Comment (FeynmanZhou) [inline docs/en/concepts-and-designs/s2i_b2i_overview.md]: I just see the introduction about S2i, without B2i in the following section, but the title is S2I/B2I overview.  @soulseen

Comment (soulseen) [reply docs/en/concepts-and-designs/s2i_b2i_overview.md]: update s2i arch.

File: docs/en/concepts-and-designs/s2i_b2i_overview.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+Besides, the S2I stack includes following CRD resources:
+
+1. s2ibuildertemplates: defines information about S2I builder image.
+2. s2ibuilders: all configuration information used in building are stored in this CRD.
+3. s2iruns: defines an action about build
+
+Here is a Architecture to figout relationship about all CRD:
+
+​	![](../../images/s2i_arch.png)
```
Comment (FeynmanZhou) [inline docs/en/concepts-and-designs/s2i_b2i_overview.md]: the image url looks incorrect.

File: docs/en/concepts-and-designs/s2i_b2i_overview.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+1. S2I Operator: A Kubernetes CRD controller that provides easy Kubernetes-style resources for declaring CI/CD-style pipelines
+2. S2IRun: It is the instance that executes the s2i build process in containers.
+
+Besides, the S2I stack includes following CRD resources:
+
+1. s2ibuildertemplates: defines information about S2I builder image.
+2. s2ibuilders: all configuration information used in building are stored in this CRD.
+3. s2iruns: defines an action about build
+
+Here is a Architecture to figout relationship about all CRD:
```
Comment (FeynmanZhou) [inline docs/en/concepts-and-designs/s2i_b2i_overview.md]: Here is a graph that describes the relationship about all CRD within S2I architecture:

File: docs/en/concepts-and-designs/s2i_b2i_overview.md (hunk 2 above)
Comment (soulseen) [reply docs/en/concepts-and-designs/s2i_b2i_overview.md]: looks well in https://github.com/soulseen/kubesphere/blob/s2i-doc/docs/en/concepts-and-designs/s2i_b2i_overview.md

**Pull request merged**